universal-application-tool-0.0.1/app/services/applicant/ReadOnlyApplicantProgramServiceImpl.java

```diff
@@ -93,4 +94,33 @@ public boolean preferredLanguageSupported() {
         .map(blockDefinition -> new Block(blockDefinition.id(), blockDefinition, applicantData))
         .collect(toImmutableList());
   }
+
+  @Override
+  public ImmutableList<AnswerData> getSummaryData() {
+    ImmutableList.Builder<AnswerData> builder = new ImmutableList.Builder<AnswerData>();
+    // TODO: Change to getBlockList() when available.
+    ImmutableList<Block> blocks = getCurrentBlockList();
```

Comment:
(blocking) `getAllBlocks` is now available!

Reply:
Done!